tube/etl/indexers/base/lambdas.py

```diff
@@ -6,25 +6,15 @@
 def extract_metadata(str_value):
     """
     Get all fields in _props (strs[3] in hadoop file) and node_id field (strs[4])
-    :param str_value:
+    :param str_value: plaintext dump of one row in HDFS file
     :return:
     """
-    origin_value = str_value
-    str_value = str_value.replace("'", "###")
-    str_value = str_value.replace('\\""', "##")
-    strs = ast.literal_eval(str_value.replace('""', "'"))
+    _, __, ___, props_text, node_id = ast.literal_eval(str_value)
     try:
-        props = json.loads(
-            strs[3].replace("'", '"').replace("###", "'").replace("##", '\\"'),
```

Comment:
looks fine to me but we might want to ask @thanh-nguyen-dang and @mfshao what this encoding was about and if removing it would cause issues (since they seem to know what's going on there, see https://github.com/uc-cdis/tube/pull/117)

Reply:
If I remember correctly, this is to address some issue caused by nested escaped `"` inside a string

Reply:
But I would hope @thanh-nguyen-dang can confirm

Reply:
yes, there is problem with the nested escaped \\" from client input @mcannalte @mfshao

Reply:
could we add a unit test for it?